settingtypes.txt

```diff
@@ -85,6 +85,11 @@ pipeworks_use_real_entities (Use Real Entities) bool true
 #A value 0.2 or above may cause issues with accelerator tubes.
 pipeworks_entity_update_interval (Entity Update Interval) float 0 0 0.8
 
+# Use the default rules from the digilines mod.
+# If enabled pipeworks devices (filter injector, deployer, dispenser, node breaker, 
+# teleport tube, lua tube, etc) will conduct digiline signals vertically.
```

Comment:
Could maybe verify what it actually affects if going to list those, as far as I know this doesn't affect lua tube for example. All other things might be affected but could either verify or leave listings out.

If going to keep listing and including vague "etc" then better to list known exceptions like lua tube and others if any.

Reply:
I conducted the testing, and modified the comment. It now reads:

# Use the default rules from the digilines mod.
# If enabled the following devices will connect to digiline networks in the vertical direction:
# digiline filter injector, deployer, dispenser, node breaker, autocrafter
# If disabled, the devices will not be able to send or recieve digiline signals from the top
# or bottom faces, regardless of the node rotation. This breaks expected behavior with digiline 
# conducting tubes, so it is recommended to enable this option unless you have specific builds
# that make use of the lack of vertical digiline connectivity and those are more important to you.